### DISCUSSION TITLE
Developers' Discussion

### DISCUSSION THREAD

**Discussion by lingbopro** [lingbopro/soberjs-vscode#5] [Ideas]
讨论一下 `v0.3.0`-`v1.0.0` 的开发路线，问题，以及废话（
@Minemetero

**Comment by Minemetero** [comment]
是的

**Reply by Minemetero** [reply]
我发布了预发布版的0.3.0

**Reply by Minemetero** [reply]
0.3.0我们先Debug现在的扩展，1.0.0把CSS给补全了就差不多了

**Reply by lingbopro** [reply]
em
我还要修一下Actions
绝对不能手动改版本号

**Reply by Minemetero** [reply]
ababaabababa

**Comment by lingbopro** [comment]
![Screenshot_20240710_095309_com.github.android.jpg](https://github.com/lingbopro/soberjs-vscode/assets/148617407/ba5ccd61-155c-4f6d-b28f-30d757ed75b0)

《在不审查的情况下合并拉取请求》

**Reply by Minemetero** [reply]
hahahah非常有趣

**Reply by Minemetero** [reply]
自杀式行为

**Reply by Minemetero** [reply]
侮辱性极强的badge

**Comment by lingbopro** [comment]
\*我得到了启发\*

![Screenshot_20240710_100902.jpg](https://github.com/lingbopro/soberjs-vscode/assets/148617407/b14e79be-20e2-41d7-873b-a9aea765b014)

**Reply by Minemetero** [reply]
那可太棒了

**Reply by lingbopro** [reply]
我明白要怎么做了
不过还得`setup-node`

**Reply by Minemetero** [reply]
abababababbabab

**Comment by lingbopro** [comment]
我发新测试版了

**Comment by lingbopro** [comment]
![image](https://github.com/lingbopro/soberjs-vscode/assets/148617407/8c822610-a6e0-468c-b46c-9cec3b6fc054)

我的第一个成功工作流！！！（激动

**Reply by Minemetero** [reply]
站在巨人~我~的肩膀上

**Reply by lingbopro** [reply]
emm

**Comment by Minemetero** [comment]
我觉得功能展示的功能可以加一下，就录制个视频，简单

**Reply by lingbopro** [reply]
不，gif

**Reply by Minemetero** [reply]
e

**Comment by lingbopro** [comment]
我感觉可以直接上`v0.3.0`正式版了

**Comment by lingbopro** [comment]
我的计划是`v0.3`完成所有组件，`v0.4`完成CSS补全，`v0.5`完成图标SVG补全，然后打磨打磨就可以上`v1.0.0`了

**Reply by Minemetero** [reply]
e好吧

**Comment by lingbopro** [comment]
那我就发v0.3.0的正式版了

**Reply by Minemetero** [reply]
好

**Reply by Minemetero** [reply]
记得要rebase and merge这样就只算一条commit

**Reply by Minemetero** [reply]
然后你把dev删了再把dev加回来，这样就没有什么behind和ahead的提示

**Reply by lingbopro** [reply]
> 记得要rebase and merge这样就只算一条commit

不是squash吗
压缩合并

**Comment by lingbopro** [comment]
v0.3.0搞定

**Reply by Minemetero** [reply]
CSS开始开发！

**Reply by lingbopro** [reply]
关于图标SVG补全我已经有一些进展了

**Reply by Minemetero** [reply]
e

**Reply by Minemetero** [reply]
其实我不知道为啥要SVG补全

**Reply by Minemetero** [reply]
我以前都没见过这个功能

**Reply by lingbopro** [reply]
就是可以快速的输入Sober图标库里的SVG，省得手动去搜索和复制了

**Reply by Minemetero** [reply]
那这工作量是不是太大了

**Reply by Minemetero** [reply]
堪比字典了

**Reply by Minemetero** [reply]
v0.2.0其实不算什么更新，顶多算个v0.1.1

**Reply by lingbopro** [reply]
不大，我已经研发出了脚本

**Reply by lingbopro** [reply]
> v0.2.0其实不算什么更新，顶多算个v0.1.1

um
确实

**Reply by Minemetero** [reply]
> 不大，我已经研发出了脚本

开个新分支，把脚本推上给我看看

**Reply by Minemetero** [reply]
或者你开个Private的repo分享给我，内部仓库 :)

**Reply by lingbopro** [reply]
其实就是一段JS而已

**Reply by lingbopro** [reply]
我可以直接发这

**Reply by Minemetero** [reply]
> 我可以直接发这

发吧

**Comment by lingbopro** [comment]
你开始搞CSS了吗
要是还没就我来罢

**Reply by Minemetero** [reply]
我搞过没搞成功

**Reply by Minemetero** [reply]
按模板来的（

**Reply by Minemetero** [reply]
你搞吧

**Reply by lingbopro** [reply]
那我来逝世

**Reply by Minemetero** [reply]
试试就逝世

**Comment by lingbopro** [comment]
我发现Sober有这么多颜色相关的CSS变量，但是官方文档一个没提...
我要不要手动写注释然后给加进扩展里去

![image](https://github.com/lingbopro/soberjs-vscode/assets/148617407/eee77d2c-c39c-4981-8b97-9891d37c33cd)

**Reply by lingbopro** [reply]
![image](https://github.com/lingbopro/soberjs-vscode/assets/148617407/fcd425c0-af24-4b66-b712-ab5b582e024f)

文档倒是提到了，但是没中文名...

**Reply by Minemetero** [reply]
这正是我想要的英语:)

**Comment by lingbopro** [comment]
我发`v0.4.0`的预发布版了

**Comment by lingbopro** [comment]
![Screenshot_20240713_095717_com.github.android.jpg](https://github.com/user-attachments/assets/6793e308-a1ae-4027-8068-5dc097f9729a)

现在我也有这个成就了

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/e5c500b4-0bd7-42ec-b0d0-b39bc313f8cd)

@Minemetero 又来活了

**Reply by Minemetero** [reply]
欸？

**Reply by Minemetero** [reply]
最经效率还不错啊跳到4.1了

**Reply by Minemetero** [reply]
这次更新谁来做？你做还是我做？

**Reply by lingbopro** [reply]
> 这次更新谁来做？你做还是我做？

都行，反正我还没开始做

**Comment by Minemetero** [comment]
commit好难看啊

**Reply by lingbopro** [reply]
这破工作流…

**Reply by lingbopro** [reply]
终于修好了！

（但是只修好了一半

**Reply by Minemetero** [reply]
太TM难看了啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊

**Reply by Minemetero** [reply]
我好难受啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊啊

**Reply by lingbopro** [reply]
没事，回头本地把提交历史改一下然后force push上来就行了（）

**Reply by lingbopro** [reply]
虽然一般不用force push，但是这显然不是一般情况（）

**Reply by Minemetero** [reply]
6

**Comment by lingbopro** [comment]
![Screenshot_20241011_211658_com.zoho.mail.jpg](https://github.com/user-attachments/assets/d0aae4f8-7e49-488c-984f-c76c71d3ca8d)

oh?

**Reply by Minemetero** [reply]
:)

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/b549ac9b-6789-420a-a202-e12525467fe7)

ooooo nice :)

**Reply by Minemetero** [reply]
6

**Comment by Minemetero** [comment]
该更新啦

**Reply by lingbopro** [reply]
不想更 :(

**Reply by Minemetero** [reply]
那我试试。。。

**Comment by Minemetero** [comment]
![image_2025-02-01_100708768](https://github.com/user-attachments/assets/37cb7e58-a556-4e2d-b4df-cd199cbe6c98)
我觉得非常有必要更新了，这么多用户。。。

**Comment by Minemetero** [comment]
https://www.vsixhub.com/vsix/142972/ 则是什么

**Reply by lingbopro** [reply]
vsixhub是什么东西

看起来这个网站从vs marketplace爬取的扩展信息
